icons/butterfly.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16 2c-4 2-4 7-4 7s0-5-4-7" />
+  <path d="M12 9v11" />
+  <path d="M20 5c-3.5 0-6.5 3.9-8 6.3C10.5 8.9 7.5 5 4 5a2 2 0 0 0-2 2c0 2.3.6 4.4 1.5 5.6C4 13.5 4.9 14 6 14h2c-.9.4-2.1.9-2.6 1.5-1.6 1.6-.9 3.4.7 4.9 1.6 1.6 3.4 2.3 4.9.7.3-.3 1-1.1 1-1.1s.6.8 1 1.1c1.6 1.6 3.4.9 4.9-.7 1.6-1.6 2.3-3.4.7-4.9-.5-.5-1.7-1.1-2.6-1.5h2c1.1 0 2-.5 2.5-1.4.9-1.2 1.5-3.3 1.5-5.6a2 2 0 0 0-2-2" />
```

Comment:
Super cute, although the antennae are a bit dense at 100%.

On high dpi yours definitely looks better, still, maybe we should increase that gap by a little?
![image](https://github.com/lucide-icons/lucide/assets/17746067/f6098ef7-5182-4019-9ed9-12c74dd4cd31)
![image](https://github.com/lucide-icons/lucide/assets/17746067/45713c7b-7f02-4839-8982-becdbe75fb8e)

```suggestion
  <path d="M16 2a4 4 0 0 0-4 4 4 4 0 0 0-4-4" />
  <path d="M12 6v14" />
  <path d="M12 10c1.5-2.5 5.5-5 8-5a2 2 0 0 1 2 2c0 2.3-.6 4.4-1.5 5.6-.5.9-1.4 1.4-2.5 1.4h-2c.9.4 2.1 1 2.6 1.5 1.6 1.5.9 3.3-.7 4.9-1.5 1.6-3.3 2.3-4.9.7-.4-.3-1-1.1-1-1.1s-.7.8-1 1.1c-1.5 1.6-3.3.9-4.9-.7-1.6-1.5-2.3-3.3-.7-4.9.5-.6 1.7-1.1 2.6-1.5H6c-1.1 0-2-.5-2.5-1.4C2.6 11.4 2 9.3 2 7a2 2 0 0 1 2-2c2.5 0 6.5 2.5 8 5" />
```

Reply:
Sneak peek of the low dpi preview, here with 18px which is commonly used in combination with text.

![](https://github.com/lucide-icons/lucide/assets/25524993/61b2b61f-f5bf-4001-b63d-3ddf4c09502c)

Looks pretty great IMO 💪

Reply:
> Sneak peek of the low dpi preview

@jguddas Nice, we definitely need to add this to the preview bot! Maybe I will get around to it eventually if no-one else does 😅

> On high dpi yours definitely looks better

@karsa-mistmere Shouldn’t we try to strike the right balance between high DPI—the vast majority of screens at this point—vs the lowest common denominator?

> maybe we should increase that gap by a little?

Yeah, although that might be a bit much; See latest `commit`… Aiming for somewhere in between. Again need that process to quickly see what it looks like at low res…

Reply:
> @karsa-mistmere Shouldn’t we try to strike the right balance between high DPI—the vast majority of screens at this point—vs the lowest common denominator?

[Citation needed.](https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide)

Reply:
> [Citation needed.](https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide)

@karsa-mistmere Interesting. I probably have a slightly privileged western bias… 😅